src/main/java/juja/microservices/keepers/dao/KeepersRepository.java

```diff
@@ -42,34 +44,44 @@
     public String save(KeeperRequest keeperRequest) {
         Date startDate = Date.from(LocalDateTime.now().atZone(ZoneId.systemDefault()).toInstant());
         Keeper keeper = new Keeper(keeperRequest.getFrom(),
```

Comment:
agree, i didn't change cause its not my task(KPR-F1). i could move it into service but this may break some tests. Could @Solykk fix his part?